app/views/layouts/shared/_footer.html.haml

```diff
@@ -7,5 +7,5 @@
         = image_tag 'facebook.png', class: 'facebook'
     %p
       = t('landing.footer.built_by')
-      = succeed ', 2018' do
+      = succeed ', 2019' do
```

Comment:
Can we use ruby to dynamically find the current year?

Reply:
I was thinking about that during the coding...So, yes I will switch to ruby also that.